### PR TITLE
Documentation for setting a Privatenet with Neo-Cli

### PR DESCRIPTION
**Pull request by Thacryba**
On [Neo Docs](https://docs.neo.org/docs/en-us/network/private-chain/private-chain2.html) there is a tutorial of how to set a Privatenet using Neo-GUI, but it doesn't have a tutorial using Neo-Cli.

Since the Neo-GUI isn't complete, I think it is important to show the steps using Neo-Cli. 

I put this tutorial on GitHub, but if you like, I can put it on Neo Docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,5 +102,9 @@ In order to synchronize the network faster, please check [here](https://docs.neo
 
 ## Usage
 
-For more information about these two nodes, you can refer to [documentation](https://docs.neo.org/docs/en-us/node/introduction.html) to try out more features. 
+See [documentation](https://docs.neo.org/en-us/node/cli/cli.html). E.g. try `show state` or `create wallet wallet.json`.
 
+
+## Seting up a privatenet using Neo-Cli
+
+Check out [this tutorial](neo-cli/Privatenet-Neo-Cli-Tutorial/set-privatenet-neo-cli.md) to create a privatenet.
```
Comment (shargon) [inline README.md]: `Neo-Cli` paths in lowercase?

**Comment by erikzhang** [comment]
Format

File: README.md (hunk 1 above)
Comment (Thacryba) [reply README.md]: Sorry, I didn't understand. Are you referring to "neo-cli" folder in lowercase or "Privatenet-Neo-Cli-Tutorial" should be in lowercase?

Comment (shargon) [reply README.md]: The folder, think that it should be

`Check out [this tutorial](neo-cli/privatenet-neo-cli-tutorial/set-privatenet-neo-cli.md) to create a privatenet.
`
or `Check out [this tutorial](neo-cli/Privatenet-neo-cli-tutorial/set-privatenet-neo-cli.md) to create a privatenet.
`

Comment (Thacryba) [reply README.md]: Ok, paths and folders fixed already.
But I can't pass on the dotnet format. I run it in my pc and it worked, but here the test always fails and I don't know why.

Comment (shargon) [reply README.md]: Also you need to move the images folders

Comment (Thacryba) [reply README.md]: Where do you think is better to put the images?

Comment (Thacryba) [reply README.md]: @shargon I was thinking about putting those images on a different repository used only for storing them. So we can take off the screenshot folder.
What do you think?

Comment (shargon) [reply README.md]: I prefer in the same repository.

Comment (Thacryba) [reply README.md]: @shargon I understand that I should move the images from the screenshot folder into the privatenet-neo-cli-tutorial folder. Did I understand that correctly?

Comment (shargon) [reply README.md]: I think that this change it's already made https://github.com/neo-project/neo-node/pull/538/commits/d9ebce97f18d53b516b38c6109e347fe113673c2

**Comment by erikzhang** [comment]
I think tutorials should be moved to https://github.com/neo-project/docs.

**Comment by Thacryba** [comment]
> I think tutorials should be moved to https://github.com/neo-project/docs.

@erikzhang I was thinking about modifying the Privatenet Tutorial on Neo Docs and include the option to set it using Neo-Cli, since there's only the option using Neo-GUI. What do you think?

**Comment by erikzhang** [comment]
> I was thinking about modifying the Privatenet Tutorial on Neo Docs and include the option to set it using Neo-Cli, since there's only the option using Neo-GUI. What do you think?

I think it's good. Then I will close this PR first.

**Pull request closed without merging**